### DISCUSSION TITLE
更新到3.31版本后，挂载天翼云盘，输入验证码，总是说错误

### DISCUSSION THREAD

**Discussion by ppp1974** [AlistGo/alist#6129] [General]
请问怎么解决啊？老是说验证码错误。

![微信截图_20240301191031](https://github.com/alist-org/alist/assets/125720190/17bbaac7-52d7-468f-97c3-e8cab358f6d3)
 
进入编辑输入验证码，每次都显示错误。

**Comment by ppp1974** [comment]
请问有解决办法吗？

**Comment by captain-lu-A320** [comment]
同样蹲天翼云盘

**Comment by fddc** [comment]
文档有说明，你们试试
https://alist.nn.ci/zh/guide/drivers/189.html
